cmd/preflight/cmd/experimental.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// experimentalCmd returns a Cobra command containing experimental subcommands.
+// These subcommands are not supported for production workflows.
+func experimentalCmd() *cobra.Command {
+	expCmd := &cobra.Command{
+		Use:     "experimental",
```

Comment:
any reason you went with exp or experimental instead of alpha ?

Reply:
That's a great question. I considered the same.

In my mind, Alpha meant it may get promoted to be a full command, and conveys at least a marginal amount of API "similarity" (e.g. it may look like this).

With the experimental nomenclature, I intended to convey that this will likely not look this way in the future. It's entirely likely if/when this gets promoted to a full command, it'll look very different from how it looks in experimental.

Reply:
This is my take, `alpha` is for something that will turn into a an exact match of a `cmd` once the underlying code is stable. This isn't that case, the underlying code lives somewhere else and is already stable and feature full, and when it's shifted `into` preflight the cmd will totally change to fit our model/style/flow, so to speak.

Reply:
`alpha` has no stability guarantees per k8s conventions .. but its a nit exp should be fine 👍